docs/GettingStarted.md

```diff
@@ -3,126 +3,43 @@
-     - [Distributed tracing](#distributed-tracing)
-     - [HTTP request queuing](#http-request-queuing)
-     - [Processing pipeline](#processing-pipeline)
-         - [Filtering](#filtering)
-         - [Processing](#processing)
-     - [Trace correlation](#trace-correlation)
-     - [Configuring the transport layer](#configuring-the-transport-layer)
-     - [Metrics](#metrics)
-         - [For application runtime](#for-application-runtime)
-     - [OpenTracing](#opentracing)
-
-## Compatibility
-
-**Supported Ruby interpreters**:
-
-| Type  | Documentation              | Version | Support type                         | Gem version support |
-| ----- | -------------------------- | -----   | ------------------------------------ | ------------------- |
-| MRI   | https://www.ruby-lang.org/ | 2.6     | Full                                 | Latest              |
-|       |                            | 2.5     | Full                                 | Latest              |
-|       |                            | 2.4     | Full                                 | Latest              |
-|       |                            | 2.3     | Full                                 | Latest              |
-|       |                            | 2.2     | Full                                 | Latest              |
-|       |                            | 2.1     | Full                                 | Latest              |
-|       |                            | 2.0     | Full                                 | Latest              |
-|       |                            | 1.9.3   | Maintenance (until August 6th, 2020) | < 0.27.0            |
-|       |                            | 1.9.1   | Maintenance (until August 6th, 2020) | < 0.27.0            |
-| JRuby | http://jruby.org/          | 9.1.5   | Alpha                                | Latest              |
-
-**Supported web servers**:
-
-| Type      | Documentation                     | Version      | Support type |
-| --------- | --------------------------------- | ------------ | ------------ |
-| Puma      | http://puma.io/                   | 2.16+ / 3.6+ | Full         |
-| Unicorn   | https://bogomips.org/unicorn/     | 4.8+ / 5.1+  | Full         |
-| Passenger | https://www.phusionpassenger.com/ | 5.0+         | Full         |
-
-**Supported tracing frameworks**:
-
-| Type        | Documentation                                   | Version               | Gem version support |
-| ----------- | ----------------------------------------------- | --------------------- | ------------------- |
-| OpenTracing | https://github.com/opentracing/opentracing-ruby | 0.4.1+ (w/ Ruby 2.1+) | >= 0.16.0           |
-
-*Full* support indicates all tracer features are available.
-
-*Deprecated* indicates support will transition to *Maintenance* in a future release.
+## Quick Start
 
-*Maintenance* indicates only critical bugfixes are backported until EOL.
+- [Install Agent][1] 
+- [Install Client][2]  
```

Comment:
Not sure if `Install Client` is the most correct way to describe installing `dd-trace-rb`.
I can think of a few suggestions: `Install Gem`, `Install Library`, `Install Tracer Library`, `Install Tracer`.

Reply:
We do sometimes refer to the tracer as "client library".

Whatever we pick should be consistent with other docs 👍

Reply:
Makes sense. With that in mind, I'd recommend `Install Client Library` then, instead of `Install Client`, if this reads better to anyone else too.